go/stats/counter.go

```diff
@@ -52,6 +52,11 @@ func (v *Counter) Add(delta int64) {
 	v.i.Add(delta)
 }
 
+// Set overwrites the current value.
+func (v *Counter) Set(value int64) {
+	v.i.Set(value)
+}
+
```

Comment:
for counter we should call add function, set is used for guage.

Reply:
@GuptaManan100 and I were originally of this opinion as well. Because we are not incrementing this stat, but reading it from MySQL, it only made sense to make it a Gauge. However, Prometheus handles well-known metrics in specific ways. It is doing a substring match for `innodb_rows` and interpreting all of them as counters.
So yeah this is non-intuitive but needed for the metrics from vttablet to work correctly with prom.

Reply:
I suppose we could rename it to something that doesn't match `innodb_rows`.